turtle_tf2_cpp/src/turtle_tf2_message_filter.cpp

```diff
@@ -52,7 +52,9 @@ class PoseDrawer : public rclcpp::Node
     tf2_listener_ =
       std::make_shared<tf2_ros::TransformListener>(*tf2_buffer_);
 
-    point_sub_.subscribe(this, "/turtle3/turtle_point_stamped");
+    point_sub_.subscribe(
+      this, "/turtle3/turtle_point_stamped",
+      rclcpp::QoS(rclcpp::QoSInitialization::from_rmw(rmw_qos_profile_default)));
```

Comment:
Especially given that this is one of our examples, I think this is a little wordy.  We can simplify this to:

```suggestion
    point_sub_.subscribe(this, "/turtle3/turtle_point_stamped", rclcpp::QoS(10));
```

(this probably points out a deficiency in our QoS APIs for this, but at least the above doesn't look as scary)